full/src/test/java/apoc/custom/CypherProceduresStorageTest.java

```diff
@@ -55,7 +58,7 @@
 public class CypherProceduresStorageTest {
     private final static String QUERY_CREATE = "RETURN $input1 + $input2 as answer";
     private final static String QUERY_OVERWRITE = "RETURN $input1 + $input2 + 123 as answer";
-    private final int refreshTime = 3000;
+    private final int refreshTime = 50;
```

Comment:
why this?

Reply:
To make the `*WorksCorrectlyWithoutConflicts()` tests more deterministic, since the bug occurs when calling a custom procedure/function during the refresh phase. 
With a high `refreshTime` the tests are almost always green even without the `synchronized` clauses.